.github/workflows/frontend:erc20-messaging.yml

```diff
@@ -7,6 +7,7 @@ on:
   workflow_dispatch:
     inputs:
       e2e-test-type:
+        required: true
```

Comment:
I don't think that's needed, right?

Reply:
we need it in order to have the default value set during `branch` event.

Reply:
Can you add a comment to that effect?

Reply:
@Freyskeyd Isn't [this change](https://github.com/topos-protocol/e2e-tests/pull/32#discussion_r1444417959) that I had proposed initially what you really want here? How adding a `required` in `workflow_dispatch`'s inputs can impact what is used by the `pull_request` event? (I assume that's the event you were referring to).

And I'm realizing that that proposed change was indeed required.

Reply:
yes indeed we can maybe simplify this